src/components/Scroller.tsx

```diff
@@ -0,0 +1,24 @@
+import { HTMLProps } from 'react'
+
+interface ScrollerProps {
+  scrollbars?: 'vertical' | 'horizontal' | 'both'
+}
+
+export function Scroller({
+  scrollbars = 'both',
+  ...props
+}: ScrollerProps & HTMLProps<HTMLDivElement>) {
+  return (
+    <div
+      {...props}
+      style={{
```

Comment:
I've tested replacing the `ScrollArea` component in other parts of the app and most of them worked, but some didn't such as `SettingsDialog` and `ChecksSection`. For those, setting `height: 100%` here did the trick.

Is this something we'd want to make default?

Reply:
+1 for `ScrollView`.

Reply:
> Is this something we'd want to make default?

I'd say not at this stage. In my testing, it worked fine in SettingsDialog, but I needed to use `asChild` in one of the Radix components, so it's not exactly a 1-1 replacement